proto/raft_serverpb.proto

```diff
@@ -23,6 +23,9 @@ message RaftMessage {
     // If it has value, to_peer should be removed if merge is never going to complete.
     metapb.Region merge_target = 9;
     ExtraMessage extra_msg = 10;
+    // use for stale read
+    uint64 applied_index = 11;
```

Comment:
Why is it necessary?

Reply:
To forward follower's `safe_ts` through raft heartbeat